scheduler/route.go

```diff
@@ -23,11 +26,16 @@ func (s scheduler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 // NewHandler return new http.Handler of the scheduler extender
-func NewHandler(divisor float64) (http.Handler, error) {
-	if divisor <= 0 {
-		return nil, fmt.Errorf("invalid divisor: %f", divisor)
+func NewHandler(defaultDiv float64, divisors map[string]float64) (http.Handler, error) {
+	if defaultDiv <= 0 {
+		defaultDiv = defaultDivisor
+	}
```

Comment:
I think defaulting should be done at configuration file parsing, not here.

Reply:
I find it difficult to do.
Because topolvm-scheduler doesn't know all the list of device classes.